docs/text/how_to_contribute.rst

```diff
@@ -64,21 +64,32 @@ you have to:
 
 .. code::
 
-    pytest
+    make test-local
```

Comment:
With this, users will need to reinstall the pip packages for testing on very run of the test. I know, it should be quick as all packages are already there, but it does take some extra time nevertheless. 

I would suggest to not add a makefile command for the local testing but to state that the users need to run pytest. The makefile command just replaced two commands and the installation command is explained on the page already anyways